cross/hoverkite-firmware/src/motor.rs

```diff
@@ -64,10 +74,71 @@ pub struct Motor {
     power: i16,
     /// The number of timer cycles since the motor power was last changed.
     smoothing_cycles: u32,
+    _emergency_off: PB12<Alternate<AF2>>,
```

Comment:
comment about this being a placeholder and wanting to push this into Pwm at some point in the future?

Reply:
Done.